### ISSUE TITLE
phoenix branch - limited to S16LE

### ISSUE THREAD

**Issue by taospartan** [introlab/odas#118]
I'm running experiments with odas phoenix branch in regards altering the bit rate from S16LE to S32LE @48K odas seems to break and just produces noise, is the phoenix branch just limited S16LE? It works perfectly when configed  to 48K @ S16LE. 

Many thanks

**Comment by FrancoisGrondin** [comment]
Yes I did limit it to S16LE for simplicity. I know it's a bummer. I plan on fixing that on the  new release ;)
